.github/workflows/pr.yaml

```diff
@@ -14,6 +14,19 @@ jobs:
   builds:
     uses: ./.github/workflows/build.yaml
 
+  verify-messages:
```

Comment:
Can you add this to format instead so we aren't stuck waiting for another machine? Real builds already take much more time anyway...

Reply:
Sounds good!